ext/dcos-installer/dcos_installer/action_lib/__init__.py

```diff
@@ -41,6 +41,21 @@ def run_preflight(config, pf_script_path='/genconf/serve/dcos_install.sh', block
     pf = get_async_runner(config, targets, async_delegate=async_delegate)
     chains = []
 
+    binary_chain = ssh.utils.CommandChain('preflight_compatibility')
+
+    # for Ubuntu/Debian derivatives
+    binary_chain.add_execute(
```

Comment:
Ideally this would execute only on Ubuntu derivatives of a specific release. I would rather this end up in a separate, Debian or Ubuntu only preflight. Generally speaking, we're about to implement a big refactor on action_lib. I think it would be nice to break out distro-specific code into their own code-bases (preflight-debian.py?) and we have a top level caller which would execute the correct action based on distro.